submodules/storage/outputs.tf

```diff
@@ -16,12 +16,19 @@ output "s3_buckets" {
   } }
 }
 
-output "s3_policy" {
-  description = "s3 IAM Policy arn"
-  value       = aws_iam_policy.s3.arn
+output "iam_policies" {
+  description = "IAM Policy ARNs"
+  value       = [aws_iam_policy.s3.arn, aws_iam_policy.ecr.arn]
```

Comment:
Shouldnt this be a dict so we know which is which?

Reply:
In this case the policies are blindly attached, do we need to know where they came from?